packages/renderer/src/lib/preferences/PreferencesRegistriesRendering.svelte

```diff
@@ -3,6 +3,7 @@ import { onMount } from 'svelte';
 import type { Registry } from '@tmpwip/extension-api';
 import { registriesInfos } from '../../stores/registries';
 import PreferencesRegistriesCreateRegistryModal from './PreferencesRegistriesCreateRegistryModal.svelte';
+import PreferncesEditRegistrieModal from './PreferncesEditRegistrieModal.svelte';
```

Comment:
the filename has a typo (missing e in Preferences) and missing s at `RegistrieS`

Reply:
> FYI I did rebase your branch
> 
> ```
> $ git remote add upstream https://github.com/containers/podman-desktop
> $ git fetch upstream
> $ git pull --rebase upstream main
> $ git push -f origin editmode
> ```

@benoitf  hey I fixed it. Which brunch should I push my code to, Main or editmode.

Reply:
always push to `editmode` (this is a the branch used to create this Pull Request)